content/en/docs/setup/platform-setup/kops/index.md

```diff
@@ -8,6 +8,10 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
+{{< tip >}}
+No special configuration is required to run Istio on clusters managed by Kops since version 1.22 running Kuberentes 1.22. For prior versions, you will need to continue to perform these steps.
```

Comment:
Not sure why Kubernetes version got in here. Maybe that is correct. It is misspelled however.

Reply:
Should `running Kubernetes 1.22` just be removed? Or is Kops configuration still needed on older Kubernetes versions?

Reply:
It doesn't have to do with the version of kops, but with kubernetes afaict.

Reply:
```suggestion
No special configuration is required to run Istio on clusters managed by Kops since version 1.22 running Kubernetes 1.22. For prior versions, you will need to continue to perform these steps.
```